apps/meteor/client/components/AutoCompleteDepartment.tsx

```diff
@@ -51,29 +51,26 @@ const AutoCompleteDepartment = ({
 	const sortedByName = useMemo(
 		() =>
 			departmentsItems.sort((a, b) => {
-				const rankA = 'name' in a ? a.name : '';
-				const rankB = 'name' in b ? b.name : '';
+				const rankA = 'name' in a ? a.label : '';
+				const rankB = 'name' in b ? b.label : '';
```

Comment:
The same goes here. I feel the data is already sorted by the backend based on the name field